gear-test/src/proc.rs

```diff
@@ -204,7 +204,8 @@ where
 
     let fixture = &test.fixtures[fixture_no];
 
-    for message in &fixture.messages {
+    let empty = Vec::new();
+    for message in fixture.messages.as_ref().unwrap_or(&empty) {
```

Comment:
```suggestion
    for message in &fixture.messages.unwrap_or_default() {
```

Reply:
```
cannot move out of `fixture.messages` which is behind a shared reference
move occurs because `fixture.messages` has type `std::option::Option<Vec<sample::Message>>`, which does not implement the `Copy` trait
```

Or if like this:
```rust
    for message in fixture.messages.as_ref().unwrap_or_default() {
```
then error is
```
the trait bound `&Vec<sample::Message>: Default` is not satisfied
the trait `Default` is implemented for `Vec<T>`
```

The only one which works, is my variant, or `clone`: 
```rust
    for message in &fixture.messages.clone().unwrap_or_default() {
```